pkg/dockerregistry/server/repositorymiddleware.go

```diff
@@ -621,34 +611,6 @@ func (r *repository) getSignatures(dgst digest.Digest) ([]digest.Digest, error)
 	return signaturesGetter.GetSignatures(r.ctx, dgst)
 }
 
-// deserializedManifestFromImage converts an Image to a DeserializedManifest.
-func (r *repository) deserializedManifestFromImage(image *imageapi.Image) (*schema2.DeserializedManifest, error) {
-	var manifest schema2.DeserializedManifest
-	if err := json.Unmarshal([]byte(image.DockerImageManifest), &manifest); err != nil {
-		return nil, err
-	}
-	return &manifest, nil
-}
-
 func (r *repository) checkPendingErrors(ctx context.Context) error {
-	return checkPendingErrors(context.GetLogger(r.ctx), ctx, r.namespace, r.name)
```

Comment:
previously, we differentiated between the context held by the startup code (used to look up the logger), and the request context. not sure if that's important, but this was preserving that